### PR TITLE
Applycourses

### PR DESCRIPTION
**Pull request by ngyihkai**
Ready

### CODE REVIEW

**Comment by ngyihkai** [comment]
Ready

**Comment by ngyihkai** [comment]
undefined method `[]' for nil:NilClass. app/controllers/sessions_controller.rb:6:in `create'
Hi Sir and Miss, may I know when the user is login, render a new page for them to apply the courses?

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -1,14 +1,52 @@
-<% provide :title, t("log_in") %>
-<h1><%= t("log_in") %></h1>
+<% provide :title, t(:log_in) %>
+<h1><%= t(:course_registration) %></h1>
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
     <%= form_for :session, url: login_path do |f| %>
       <%= f.label :email, t(:email)  %>
       <%= f.email_field :email, class: "form-control" %>
+      <%= f.label :password, t(:password)  %>
```
Comment (huongnt-2545) [inline app/views/sessions/new.html.erb:8]: Line 8->10 should be:
```suggestion
      <%= f.label :password, t(:password)  %>
      <%= f.password_field :password, class: "form-control" %>
      <%= link_to "(forgot password)", password_resets_path %>
```

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -1,14 +1,52 @@
...
       <%= link_to "(forgot password)", password_resets_path %>
       <%= f.password_field :password, class: "form-control" %>
       <%= f.submit "Log in", class: "btn btn-primary" %>
     <% end %>
+
     <p><%= t "new_user?" %><%= link_to t("sign_up_now!"), signup_path %></p>
+
+    <div class="field">
```
Comment (huongnt-2545) [inline app/views/sessions/new.html.erb:16]: What is purpose for these code? from line 16 -> 49

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -5,19 +5,6 @@
   <div class="col-md-6 col-md-offset-3">
     <%= render "form" %>
```
Comment (huongnt-2545) [inline app/views/users/new.html.erb:6]: why do you define 2 form in this file?
please check the tutorial for this

**Review by huongnt-2545 — COMMENTED** [review]
1. User login vs User apply course are separately
Steps: user login system -> user go to courses page -> user choose a course -> user apply course
2. You should create courses page to list all courses are opened

File: app/views/sessions/new.html.erb (hunk 2 above)
Comment (ngyihkai) [reply app/views/sessions/new.html.erb:16]: to allow user to choose several courses that are listed there

Comment (ngyihkai) [reply app/views/sessions/new.html.erb:16]: is it in order to from log in page to course page, i need to use redirect_to @courses in courses_controller?

**Comment by ngyihkai** [comment]
Ready

**Comment by ngocvt-0484** [comment]
commented

**Pull request closed without merging**